src/components/_video/VideoThumbnail/VideoThumbnail.styles.ts

```diff
@@ -0,0 +1,135 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { cVar, square } from '@/styles'
+
+const sharedOverlayStyles = css`
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 0;
+  padding-top: 56.25%;
```

Comment:
Wouldn't `height 100%` be enough since we have position absolute here?

Reply:
I refactored this a bit, but I don't think I could just get rid of this padding-top if that's what you mean